### PR TITLE
Update development.md to mention "callable" and __init__.py

### PR DESCRIPTION
**Pull request by FLamparski**
Updating the docs after I needed to do these things for firmware v1.2.0 to get a hello world app running

### CODE REVIEW

File: docs/tildagon-apps/development.md [hunk 1]
```diff
@@ -36,6 +36,12 @@ class ExampleApp(app.App):
 __app_export__ = ExampleApp
 ```
 
+If you place this code in a file called `app.py`, you may also need to add a separate file called `__init__.py` containing this line:
+
+```python
+from .app import ExampleApp
```
Comment (hughrawlinson) [inline docs/tildagon-apps/development.md:42]: This is not necessary. If the app isn't running without this on your badge, that's a bug. The format for an app does not include the __init__.py file, so we would fix it by identifying why your badge requires that.

File: docs/tildagon-apps/development.md [hunk 2]
```diff
@@ -434,7 +441,8 @@ You can test your app on-device, without publishing it, using [`mpremote`](https
     ```json
     {
         "name": "The OG Snake app",
-        "path": "apps.snake.app"
+        "path": "apps.snake.app",
+        "callable": "SnakeApp"
```
Comment (hughrawlinson) [inline docs/tildagon-apps/development.md:445]: This isn't the preferred way of doing this - as shown on line 36, apps should define `__app_export__` and point it to their app class.

**Review by hughrawlinson — APPROVED** [review]

File: docs/tildagon-apps/development.md [hunk 3]
```diff
@@ -26,6 +25,7 @@ class ExampleApp(app.App):
             # Calling clear() ensures the next time you open the app, it stays open.
             # Without it the app would close again immediately.
             self.button_states.clear()
+            self.minimise()
```
Comment (npentrel) [inline docs/tildagon-apps/development.md:28]: Good catches, thank you!

File: docs/tildagon-apps/development.md [hunk 4]
```diff
@@ -36,6 +36,12 @@ class ExampleApp(app.App):
 __app_export__ = ExampleApp
 ```
 
+If you place this code in a file called `app.py`, you may also need to add a separate file called `__init__.py` containing this line:
+
+```python
+from .app import ExampleApp
+```
+
```
Comment (npentrel) [inline docs/tildagon-apps/development.md:44]:
```suggestion
```

File: docs/tildagon-apps/development.md [hunk 5]
```diff
@@ -420,12 +426,13 @@ You can also create your own user interfaces using the [`ctx` graphics library](
 
 You can test your app on-device, without publishing it, using [`mpremote`](https://docs.micropython.org/en/latest/reference/mpremote.html).
 
-1. Create a `metadata.json` file in your app's directory. This is necessary **only** during development. Remove this file before publishing your app to the app store.
+1. Create a `metadata.json` file in your app's directory. This is necessary **only** during development. Remove this file before publishing your app to the app store. You may also need to specify a `callable` -- this should be the same as the name of your app's main class.
```
Comment (npentrel) [inline docs/tildagon-apps/development.md:429]:
```suggestion
1. Create a `metadata.json` file in your app's directory. This is necessary **only** during development. Remove this file before publishing your app to the app store.
```

File: docs/tildagon-apps/development.md [hunk 6]
```diff
@@ -420,12 +426,13 @@ You can also create your own user interfaces using the [`ctx` graphics library](
...
 
     ```json
     {
         "name": "<app-name>",
-        "path": "apps.<folder-name>.app"
+        "path": "apps.<folder-name>.app",
+        "callable": "ExampleApp"
```
Comment (npentrel) [inline docs/tildagon-apps/development.md:435]:
```suggestion
        "path": "apps.<folder-name>.app"
```

File: docs/tildagon-apps/development.md (hunk 2 above)
Comment (npentrel) [inline docs/tildagon-apps/development.md:445]:
```suggestion
        "path": "apps.snake.app"
```

**Comment by npentrel** [comment]
@FLamparski Thanks for filing this - you were seeing bugs that your changes would indeed fix but we've since fixed the bugs, so what you were seeing is no longer necessary. I will merge your other fixes though :)

**Comment by FLamparski** [comment]
Thank you @npentrel! I didn't realise that this doubled as a bug report 😂

**Pull request merged**